.github/workflows/binaries.yml

```diff
@@ -53,9 +53,9 @@ jobs:
           - name: stellar-cli
             binary: stellar
         sys:
-          - os: macos-14
+          - os: macos-15
             target: aarch64-apple-darwin
-          - os: macos-13
+          - os: macos-14
             target: x86_64-apple-darwin
```

Comment:
14 is arm only. this needs to be 15-intel.

Reply:
https://github.com/stellar/stellar-cli/pull/2276/commits/aa8a7f3ce62a8c8246dd20832e6e97da6baff45d